spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Container.scala

```diff
@@ -41,18 +43,82 @@ abstract class Container[A <: Activity : ClassTag,
     B <: Activity : ClassTag, T: ClassTag](
   implicit ev: TensorNumeric[T]) extends AbstractModule[A, B, T] {
 
+  var parent: Container[A, B, T] = null
+
   // list of sub modules
   val modules: ArrayBuffer[AbstractModule[Activity, Activity, T]]
   = ArrayBuffer[AbstractModule[Activity, Activity, T]]()
 
+  // Return empty list as compile do nothing by default.
+  def compilingPath(): List[Node[AbstractModule[Activity, Activity, T]]] = {
```

Comment:
should it be private? Is it a public API? use java doc instead of normal comments

Reply:
Please add detail instruction of this method. It's really hard to guess what's it for from the name